src/SDKs/CognitiveServices/dataPlane/AnomalyDetector/AnomalyDetector/Microsoft.Azure.CognitiveServices.AnomalyDetector.csproj

```diff
@@ -1,22 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.AnomalyDetector</PackageId>
-		<Description>Microsoft Azure Management AnomalyDetector library</Description>
-		<AssemblyName>Microsoft.Azure.Management.AnomalyDetector</AssemblyName>
-		<Version>0.9.0-preview</Version>
-		<PackageTags>Azure AnomalyDetector;</PackageTags>
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.AnomalyDetector</PackageId>
+    <Version>0.8.0-preview</Version>
```

Comment:
I see you are bumping down the version number and changing the name. Any reason to pick 0.8.0 here instead of something else (like 0.1.0)?

Reply:
@shahabhijeet merged some project structures in https://github.com/Azure/azure-sdk-for-net/pull/5464 because he didn't aware that there already has a ready SDK PR. So I solved the conflicts after that. Please ignore all these changes.

@dsgouda suggested me to use 0.8.0-preivew in previous comments.

Reply:
@AlexGhiondea For brand new RPs we recommend starting from `0.8.0-preview` so that when they are ready to release a stable version they can start from `1.0.0`. we apply the same guidelines when working with the next preview versions (`1.8.0-preview`-`2.0.0`)